packages/schemas/src/types/log.ts

```diff
@@ -137,3 +139,11 @@ export type LogPayloads = {
 export type LogType = keyof LogPayloads;
 
 export type LogPayload = LogPayloads[LogType];
+
+export type LogDTO = Omit<Log, 'payload'> & {
```

Comment:
nit: should be `Dto` in our case? i'm ok with both cc @xiaoyijun

Reply:
created LOG-2832, will refactor later since we have other DTOs.